docs/overview.md

```diff
@@ -93,6 +93,22 @@ official code with extra functionality. This allows the Agent to give an
 experience closest to its official counterparts compared to alternatives which
 may try to reimplement everything from scratch.
 
+### Why not just use Telegraf?
+Telegraf is a fantastic project and was actually considered as an alternative to 
+building a custom Prometheus scraper and agent.
```

Comment:
Hmm, I don't want to give the impression that we've built a custom Prometheus scraper here. Maybe we should just say something like

```
as an alternative to building our own agent
```

WDYT?

Reply:
Sounds good to me!